src/main/java/org/jabref/logic/importer/PagedSearchBasedFetcher.java

```diff
@@ -0,0 +1,20 @@
+package org.jabref.logic.importer;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.paging.Page;
+
+public interface PagedSearchBasedFetcher extends SearchBasedFetcher {
+
+    Page<BibEntry> performSearchPaged(String query, int size);
+
+    Page<BibEntry> performSearchForNextPage(Page<BibEntry> currentPage);
+
+    Page<BibEntry> performSearchForPreviousPage(Page<BibEntry> currentPage);
+
+    Page<BibEntry> performSearchForJumpToPage(Page<BibEntry> currentPage, int jumptTo);
```

Comment:
Search methods for next, prev and jumpTo requires the current Page. this way the fetcher can stay stateless.